WordPress/Classes/ViewRelated/Reader/ReaderSidebarViewModel.swift

```diff
@@ -20,7 +20,7 @@ final class ReaderSidebarViewModel: ObservableObject {
     }
 
     func getTopic(for topicType: ReaderTopicType) -> ReaderAbstractTopic? {
-        return try? ReaderAbstractTopic.lookupAll(in: contextManager.mainContext).first {
+        return try? ReaderAbstractTopic.lookupAllMenus(in: contextManager.mainContext).first {
```

Comment:
This is a small optimization to ensure the app doesn't need to sift through everything.